src/pages/messages/components/NviCandidateActions.tsx

```diff
@@ -159,51 +159,55 @@ export const NviCandidateActions = ({ nviCandidate, nviCandidateQueryKey }: NviC
 
       <Box sx={{ m: '1rem' }}>
         {sortedNotes.length > 0 && (
-          <Box
-            component="ul"
-            sx={{
-              display: 'flex',
-              flexDirection: 'column',
-              listStyleType: 'none',
-              p: 0,
-              m: '0 0 1rem 0',
-              gap: '0.25rem',
-            }}>
-            {sortedNotes.map((note) => {
-              let deleteFunction: (() => Promise<void>) | undefined = undefined;
-              const noteIdentifier = note.identifier;
+          <>
```

Comment:
Stor diff fordi jeg la inn en ny `<></>`.  Dette for å legge til Divider bare når vi faktisk viser noen kommentarer

Reply:
etter diskusjon med PO og designer ble disse kommentarene flyttet ned til under kommentar-boks i stedet